frontends/web/src/routes/account/account.tsx

```diff
@@ -214,33 +221,20 @@ export function Account({
                 code={code}
                 unit={balance.available.unit} />
             )}
-            <div className="flex flex-row flex-between flex-items-center flex-column-mobile flex-reverse-mobile">
-              <label className="labelXLarge flex-self-start-mobile">
+
+            <div className="flex flex-column flex-reverse-mobile">
+              <label className="labelXLarge flex-self-start-mobile hide-on-small">
                 {t('accountSummary.availableBalance')}
               </label>
-              <div className={style.actionsContainer}>
-                {canSend ? (
-                  <Link key="sendLink" to={`/account/${code}/send`} className={style.send}>
-                    <span>{t('button.send')}</span>
-                  </Link>
-                ) : (
-                  <span key="sendDisabled" className={`${style.send} ${style.disabled}`}>
-                    {t('button.send')}
-                  </span>
-                )}
-                <Link key="receive" to={`/account/${code}/receive`} className={style.receive}>
-                  <span>{t('button.receive')}</span>
-                </Link>
-                { moonpayBuySupported && (
-                  <Link key="buy" to={`/buy/info/${code}`} className={style.buy}>
-                    <span>{t('button.buy')}</span>
-                  </Link>
-                )}
+              <div className="flex flex-row flex-between flex-item-center flex-column-mobile flex-reverse-mobile">
+                <Balance balance={balance} />
+                <label className="labelXLarge flex-self-start-mobile hide-on-large show-on-small">
```

Comment:
I'd assume `show-on-small` should take care is `hide-on-large` really needed?

Reply:
Yeah that's right. I didn't notice `.show-on-small` has `display:none` on desktop lol. Change has been made.